src/interval.jl

```diff
@@ -1,12 +1,13 @@
 """
     Interval{T, L <: Bound, R <: Bound}
 
-An `Interval` represents a non-iterable range or span of values (non-interable because,
-unlike a `StepRange`, no step is defined).
+An `Interval` represents a non-iterable range or span of values (non-interable
+because, unlike a `StepRange`, no step is defined).
 
-An `Interval` can be closed (both `first` and `last` are included in the interval), open
-(neither `first` nor `last` are included), or half-open. This openness is defined by the
-bounds information which is stored as the type parameters `L` and `R`.
+An `Interval` can be closed (both `first` and `last` are included in the
+interval), open (neither `first` nor `last` are included), or half-open. This
+openness is defined by the bounds information which is stored as the type
+parameters `L` and `R`.
```

Comment:
This repo follows the Blue style so the max line length for this repo is 92: https://github.com/invenia/BlueStyle#synopsis